src/Transformers/JsonApi/resource/normalizePaging.js

```diff
@@ -0,0 +1,28 @@
+import qs from 'qs';
+import url from 'url';
+
+/**
+ * @param {JsonApiResponse} response
+ *
+ * @returns {{count: Number, pages: Number}}
+ */
+export default function normalizePaging(response) {
+  if (!response.links.last) {
+    return {
+      count: 0,
+      pages: 0,
+    };
+  }
+
+  const query = qs.parse(
+    url.parse(response.links.last).query,
+  );
+
+  const numPages = parseInt(query.page.number, 10);
+  const numPerPage = parseInt(query.page.size, 10);
+
+  return {
+    count: numPages * numPerPage,
```

Comment:
Dit is meestal niet waar. Het is natuurlijk wel een goede benadering en we kunnen het ook niet op een andere manier eruit krijgen. Gewoon zo laten dus.